blmath/util/json.py

```diff
@@ -0,0 +1,46 @@
+'''
+If an object defines for_json(), it will be serialized as what that returns.
+    class SerializableThing(object):
+        def for_json(self):
+            return "|SerializableThing|"
+
+'''
+
+from baiji.serialization import json
+from baiji.serialization.json import JSONDecoder
+
+class BlmathJSONDecoder(JSONDecoder):
+    SUPPORTED_TYPES = {'value': 'blmath.value.Value'}
+    def __init__(self):
+        super(BlmathJSONDecoder, self).__init__()
+        self.register(self.decode_supported_types)
+
+    def decode_supported_types(self, dct):
```

Comment:
Let's clean this up some, since it's just `Value`:

```py
from blmath.value import Value

def decode_value(self, dct):
    if "__value__" in  dct.keys():
        return Value.from_json(dct)
```

We can leave it as is in core, so we have the example for the future and so that `bodylabs.serialization.json` continues to handle values as advertised. But here in measurements, let's be nice and simple.